build.gradle

```diff
@@ -249,6 +243,9 @@ publishing {
 }
 
 artifactoryPublish {
+  onlyIf {
+    System.env.'TRAVIS_BRANCH' == "master" && !System.env.'TRAVIS_PULL_REQUEST'.toBoolean()
```

Comment:
How will this affect publishing releases (presumably not done directly from master)?